sys/net/gnrc/network_layer/ipv6/nib/_nib-6lr.c

```diff
@@ -35,6 +35,24 @@ static uint8_t _update_nce_ar_state(const sixlowpan_nd_opt_ar_t *aro,
         _evtimer_add(nce, GNRC_IPV6_NIB_ADDR_REG_TIMEOUT,
                      &nce->addr_reg_timeout,
                      byteorder_ntohs(aro->ltime) * SEC_PER_MIN * MS_PER_SEC);
+        if (IS_ACTIVE(CONFIG_GNRC_IPV6_NIB_ARSM)) {
+            switch (_get_nud_state(nce)) {
+            case GNRC_IPV6_NIB_NC_INFO_NUD_STATE_UNMANAGED:
+            case GNRC_IPV6_NIB_NC_INFO_NUD_STATE_REACHABLE:
+                /* nothing to do */
+                break;
+            default: {
+                    gnrc_netif_t *netif = gnrc_netif_get_by_pid(
```

Comment:
Can `netif` be `NULL` here?

Reply:
Doesn't look like it `nce` is always associated with an interface.

Reply:
Still good point. The `netif` is already available in the caller of this function, so just provide it instead of getting it again. I also added an `assert` for good measure, but really this shouldn't be hit, as the `_handle_sl2ao()` in the caller also asserts this.